### ISSUE TITLE
Add permission fixing in finish module

### ISSUE THREAD

**Issue by saramonzon** [BU-ISCIII/buisciii-tools#212]
Although the HPC configuration should ensure that the permissions for folders is `775` and `664` for files, all of them assigned to group `bi`, somehow some intermediate or copied files are created with user specific permissions in the shared folders.
This is annoying for several reasons:
- Sometimes any other user needs to modify that file, or include some file in that "wrong permissions" folder and we need to ask to the owner to change it.
- Sometimes the owner may be not working in the unit anymore or can be on holidays so we need to create a ticket for Bruno to delete it.
- Archive module strongly fails when it tries to handle a file or folder with wrong permissions making that we need to start all over the process, which is a real pain.

The solution would be to add a process after finish module so when each service analysis is finished, we fix the permissions ensuring that  all folders are `775`, and all files `664` and adding `bi` group ownership to all files and folders. Ideally this permission and group configuration should be stablished in the config file so we can change it if needed without changing the code.

**Comment by saramonzon** [comment]
It would also be helpful to add a helper script does fix this to any folder, like references or research so we can easily run it when we create a new folder in the HPC if the permissions are wrongly set. This script must be documented in the Wiki.

**Comment by Shettland** [comment]
> It would also be helpful to add a helper script does fix this to any folder, like references or research so we can easily run it when we create a new folder in the HPC if the permissions are wrongly set. This script must be documented in the Wiki.

I agree. I think it would be more useful to create a separate script so it can be executed from finish or copy-sftp without repeating lines of code
